### PR TITLE
Censys tests

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Whitespace is tricky
So are variable types.
Try some mypy help.

Add some basic tests for the Censys gatherer, do some minor refactoring, and add some small type hints along the way.

### CODE REVIEW

File: gatherers/censys.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import json
 import csv
 import logging
+from typing import List
```
Comment (konklone) [inline gatherers/censys.py]: Will this lock us to Python 3.7+ as written?

File: gatherers/censys.py [hunk 2]
```diff
@@ -127,45 +128,61 @@ def gather(suffixes, options, extra={}):
...
-    from_clause = """
-    `censys-io.certificates_public.certificates`,
-    UNNEST(parsed.subject.common_name) AS common_names,
-    UNNEST(parsed.extensions.subject_alt_name.dns_names) AS sans
-    """
+    from_clause = "\n".join([
+        "    `censys-io.certificates_public.certificates`,",
+        "    UNNEST(parsed.subject.common_name) AS common_names,",
+        "    UNNEST(parsed.extensions.subject_alt_name.dns_names) AS sans",
+    ])
```
Comment (konklone) [inline gatherers/censys.py]: I don't have strong feelings either way, but what makes `"\n".join([ ... ])` better than using `"""..."""` syntax for long multi-line strings?

File: gatherers/censys.py (hunk 1 above)
Comment (tadhg-ohiggins) [reply gatherers/censys.py]: No, typing supported this some time back. It definitely works in my local 3.6 environment, and I'm sure it works in 3.5 as well.

File: gatherers/censys.py (hunk 2 above)
Comment (tadhg-ohiggins) [reply gatherers/censys.py]: It's much easier to see whitespace differences. With the multi-line strings you have to conform to the indentation of the surrounding block.
Using the join approach makes it very clear when you want to add space on purpose versus when you're just doing that to satisfy the parser.

**Comment by konklone** [comment]
Thanks, that addressed my questions!

**Pull request merged**